modules/localplay/upnp/upnpplayer.class.php

```diff
@@ -69,8 +69,8 @@ private function Playlist()
      */
     public function __construct($name = "noname", $description_url = "http://localhost")
     {
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpdevice.php';
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpplaylist.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPDevice.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPPlaylist.php';
```

Comment:
Why did you put some letter uppercase here? It is lowercase on the repository and could make linux os to fail.